### PR TITLE
Updating PR 287

### PR DESCRIPTION
**Pull request by GabrielBachmann**
This is a remake of PR #287 .
All changes are related to documentation of material class.

### CODE REVIEW

File: ross/rotor_assembly.py [hunk 1]
```diff
@@ -74,7 +75,7 @@ class Rotor(object):
         List with the shaft elements
     disk_elements : list
         List with the disk elements
-    bearing_seal_elements : list
+    bearing_elements : list
```
[the remark this answers is not in the document]
Comment (GabrielBachmann) [reply ross/rotor_assembly.py]: To generalize the save and load methods(for example, if added new elements, there is no need to change the Rotor.load() method), I’ve used the name of the object instantiated to create a list, transforming the BearingElement to a string called bearing_elements (with ross.utils.convert ).

**Comment by rodrigomoliveira1** [comment]
@GabrielBachmann, check this PR for conflicts with PR #432. Both work in the same code lines sometimes.

**Comment by rodrigomoliveira1** [comment]
@GabrielBachmann, can we close this PR and keep only the PR #432?

**Comment by GabrielBachmann** [comment]
Actually, the commits 8f4a17e and 5dc175a are not in #432.
I think that the best way to solve this is resolving all conflicts.

**Pull request merged**